utils/ssh.py

```diff
@@ -123,6 +123,10 @@ def __init__(self, stream_output=False, **connect_kwargs):
     def is_container(self):
         return self._container is not None
 
+    @property
+    def username(self):
+        return self._connect_kwargs.get('username', None)
```

Comment:
The `get` is returning None by default if the key was not found. So

    return self._connect_kwargs.get('username')

is sufficient.